gensim/models/lsimodel.py

```diff
@@ -221,7 +223,7 @@ def merge(self, other, decay=1.0):
 #endclass Projection
 
 
-class LsiModel(interfaces.TransformationABC):
+class LsiModel(interfaces.TransformationABC,basemodel.BaseTopicModel):
```

Comment:
same here. You should probably consider setting up a flake8 checks plugin in your editor. Will help you eliminate those nitpicks.